string answer
---
bool success
string timeElapsed